topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,130 @@
+ <br><br>
+
+
+## View annotated features in JBrowse
+
+Now that we have annotated the draft genome sequence, we would like to view the sequence in the JBrowse genome viewer. First, we have to make a JBrowse file. Then, we can view it within Galaxy.
+
+> ### {% icon hands_on %} Hands-on: Create the JBrowse file
+>
+> 1. Search for **JBrowse** {% icon tool %} and run it with the following parameters
+>    - "Reference genome to display" to `Use a genome from history`
+>    - "Select the reference genome" to `Prokka on data XX.fna`.
+>       
+>       This sequence will be the reference against which annotations are displayed
+>
+>    - "Produce Standalone Instance" to `Yes`
+>    - "Genetic Code" to `11: The Bacterial, Archaeal and Plant Plastid Code`
+>    - "JBrowse-in-Galaxy Action" to `New JBrowse Instance`
+>    - "Track Group"
+>    - We will now set up one track - each track is a dataset displayed underneath the reference sequence (which is displayed as nucleotides in FASTA format).
+>    - We will choose to display the annotations (the Prokka.gff file).
+>
+>       - **Track 1 - sequence reads**: Click on `Insert Track Group` and fill it with
+>           - "Track Cateogry" to `gene annotations`
+>           - Click on `Insert Annotation Track` and fill it with
+>               - "Track Type" to `GFF/GFF3/BED/GBK Features`
+>               - "GFF/GFF3/BED Track Data" to `Prokka on data XX:gff`
+>               - "Track Visibility" to `On for new users`
+>               - "JBrowse Track Type [Advanced]" to `Canvas Features`
+>               - Click on "JBrowse Styling Options [Advanced]"
+>               - "JBrowse style.label" to `product,name,id`
+>               - "Track Visibility" to `On for new users`
+>               - Click Execute
+>
+>  - A new file will be created in your history, this contains the JBrowse interactive visualisation. We will now > view its contents and play with it
+> <br><br>
+> {: .hands_on}
+> ### {% icon hands_on %} Hands-on: Visualize the annotation
+>
+> 2. Inspect the `JBrowse on data XX and data XX - Complete` file by clicking on the eye icon
+>
+>    The JBrowse window will appear in the centre Galaxy panel.
+>
+> 3. Display all the tracks and practice maneuvering around
+>    1. Click on the tick boxes on the left to display the tracks
+>    2. Select contig 1 in the drop down box. You can only see one contig displayed at a time.
+>    1. Zoom out by clicking on the `minus` button to see sequence reads and their coverage (the grey graph)
+>    1. Zoom in by clicking on the `plus` button to see annotations.
+>    1. JBrowse displays the sequence and a 6-frame amino acid translation.
+>    1. Right click on a gene/feature annotation (the bars on the annotation track), then select View Details to see more information.
+>      - gene name
+>      - product name
+>      - you can download the FASTA sequence by clicking on the disk icon.
+> <br><br>
+> ![JBrowse](images/jbrowse6.png)
```

Comment:
If you add a legend for the image `![JBrowse](images/jbrowse6.png "My legend")` some space is automatically add before and after. So you can remove the `<br>` (which are not really markdown... and we would like to limit no markdown in the tutorials...)